apps/web/src/lib/fetchPayloadDataRest.ts

```diff
@@ -19,12 +20,13 @@ export type FetchPayloadResponse<T> = T | { error: string };
 async function fetchPayloadDataRest<T>({
   endpoint,
   params,
+  showDraft,
   next,
   accessToken
 }: FetchPayloadRequest): Promise<FetchPayloadResponse<T>> {
   const url = `${WEB_URL}${endpoint}${qs.stringify(params, {
     addQueryPrefix: true
-  })}`;
+  })}?&draft=${showDraft}`;
```

Comment:
Question about the formatting of this URL.

Typically the first param starts after the `?`, and then subsequent params are separated by the `&` symbol.

For example, `?param1=a&param2=b` to represent `{ param1: 'a', param2: 'b'}`.

In this case we're using `qs` to stringify the `params` object, then appending `?&` to the end of those.

Should we do something like this instead?

```
const url = `${WEB_URL}${endpoint}${qs.stringify({ ...params, draft: showDraft }, {
    addQueryPrefix: true
  })}`;
```

Reply:
Good catch, updated in latest push. 👍